templates_jinja2/suggest_apiwrite.html

```diff
@@ -0,0 +1,101 @@
+{% extends "base.html" %}
+
+{% block title %}The Blue Alliance - Request Write Keys{% endblock %}
+
+{% block meta_description %}Request auth tokens to import your offseason event to The Blue Alliance{% endblock %}
```

Comment:
Oh I didn't realize that was in the meta tag. Yes, put it in the overview paragraph.